### ISSUE TITLE
Strange effect of moving google calender dates on this pageAll browsers shift calendar display dates on this page while only browsing

### ISSUE THREAD

**Issue by Brian1Gaff** [nvaccess/nvda#8776]
Go to this page in any browser using nvda  and use h to get to the events diary
www.kingstonassociationforblind.org/
Note that when you cursor down it will tell you where the start date is and cursor down once more and the result is going back a month. If you then cursor up, you find the  legend has gone back also, and if you go down again it goes back another month and so on back to April.

I did go to this same page using Jaws and it does not seem to do the same  thing, it just works.
 I have had it tested on Chrome and this also does the same.
 I myself tested it on IE 11, Firefox 62.02 and Waterfox, all with the same results.
 It seems as if nvda is somehow affecting the clickable above the calendar, which is of course from google.
 I have tried an earlier nvda stable and since my version of Firefox does not work in the last version I could not tell what it did, but all other browsers still showed the issue, so its probably been here for some time if you deem it to be an nvda issue.
 I'm using  Windows 7pro 64and all browsers tested are 64 bit. No difference if you restart turn add ons off  in ff/waterfox, or nvda that I can see.
 

I think that is all you need to know from the template.
 Please ask if you cannot duplicate it your end.
 This site is our own charitys one and the site writer is not able to work out what is wrong his end.
 Brian

**Comment by jscholes** [comment]
Direct link to the problematic Google Calendar embed:
https://calendar.google.com/calendar/embed?showNav=0&showDate=0&showPrint=0&showTabs=0&showCalendars=0&showTz=0&mode=AGENDA&height=400&wkst=1&bgcolor=%23ffff00&src=kingstonassoc@gmail.com&color=%23691426&ctz=Europe/London&pli=1

**Comment by Brian1Gaff** [comment]
Thanks, Its an awfully complex link, If you want I'll ask the site builder 
for his permission to give his email out, but he is away this weekend.
 Brian


bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Brian1Gaff** [comment]
This post taken from user list for information.
Well, as soon as NVDA reads the line saying viewing dates after, the 
calendar is adjusted. If you read get to that line and then ask NVDA to read 
the line again with NVDA-up then it will read one month back, though 
pressing NVDA-up again does not adjust the scroll. I suppose I should try 
the site out on my Macintosh, where by default mouse and focus  follow 
screen reader cursor. One minor point of interest is that the message 
indicating that only events after September 22 are being displayed is 
offscreen text. I have occasionally experienced NVDA misbehaving with links 
that are offscreen with a background image for the sighted person. Perhaps 
an offscreen clickable is being clicked when it is read by NVDA.

You might want to ask on WEB-AIM mailing list for advice, though since the 
calendar is in a frame  it makes it a little more difficult to diagnose.

Jonathan Cohn


bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by ehollig** [comment]
@Brian1Gaff is this still an issue? Strangely, I am not able to reproduce the steps as you describe. Could you provide updated steps to reproduce?

**Comment by Adriani90** [comment]
The design of the calender and the website changed significantly. @Brian1Gaff could you please post an updated test case? Thanks.

**Comment by Adriani90** [comment]
We do not have any updates from the initial author since more than 1.5 years. Closing as abandoned. Please comment on it if you still have this issue and we can reopen. Thanks.
